modules/eido/validate/main.nf

```diff
@@ -0,0 +1,37 @@
+process EIDO_VALIDATE {
+    tag '$samplesheet'
+    label 'process_low'
+
+    conda (params.enable_conda ? "conda-forge::eido=0.1.8" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://containers.biocontainers.pro/s3/SingImgsRepo/eido/0.1.9_cv1/eido_0.1.9_cv1.sif' :
+        'biocontainers/eido:0.1.9_cv1' }"
+
+    input:
+    path samplesheet
+    path schema
+
+    output:
+    path "versions.yml"           , emit: versions
+    path "conversion.out"         , emit: conversion
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    """
```

Comment:
```suggestion
    def prefix     = task.ext.prefix ?: "validation"
    """
```

I add this as in some cases people may have multiple input samplesheets (e.g. for taxprofiler: one of samples, one for 'genomes'), so it's good to add this flexibiility early on.